plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -55,10 +55,12 @@ export default function Echart({
       type: 'downplay',
       dataIndex: previousSelection.current.filter(value => !currentSelection.includes(value)),
     });
-    chartRef.current.dispatchAction({
-      type: 'highlight',
-      dataIndex: currentSelection,
-    });
+    if (currentSelection.length > 0) {
```

Comment:
Thanks for the fix! Can we simplify this a bit?
```suggestion
    if (currentSelection.length) {
```